lxd/instance/drivers/driver_lxc.go

```diff
@@ -4084,7 +4087,7 @@ func (d *lxc) Update(args db.InstanceArgs, userRequested bool) error {
 				}
 			} else if key == "security.devlxd" {
 				if value == "" || shared.IsTrue(value) {
-					err = d.insertMount(shared.VarPath("devlxd"), "/dev/lxd", "none", unix.MS_BIND, false)
+					err = d.insertMount(shared.VarPath("devlxd"), "/dev/lxd", "none", unix.MS_BIND, "")
```

Comment:
@brauner should this be `idmap.IdmapStorageNone` or `idmap.IdmapStorageUnknown` rather than empty string for consistency?

Reply:
Yep, fixed.